pdpyras.py

```diff
@@ -18,16 +18,18 @@
 else:
     string_types = basestring
 
-__version__ = '4.1.1'
+__version__ = '4.1.2'
 
 
 # These are API resource endpoints/methods for which multi-update is supported
-valid_multi_update_paths = [
+VALID_MULTI_UPDATE_PATHS = [
     ('incidents', '{index}'),
     ('incidents', '{id}', 'alerts', '{index}'),
     ('priorities', '{index}'),
 ]
 
+ITERATION_LIMIT = 1e4
```

Comment:
Can you add a quick explainer with a link to the 10k limit here please, for future lurkers :)